### PR TITLE
New LF topnav design

### PR DESCRIPTION
**Pull request by cjyabraham**

### CODE REVIEW

File: web/wp-content/themes/lfevents/template-parts/upcoming-events-loop.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+						?>
+					</p>
+
+				</div>
+
+				<?php
+			}
+			wp_reset_postdata();
+		} else {
+			echo "Sorry there aren't any events currently scheduled.";
```
Comment (cjyabraham) [inline web/wp-content/themes/lfevents/template-parts/upcoming-events-loop.php:121]: Why isn't this currently showing [here](https://pr-608-lfasiallcci.pantheonsite.io/)?  And can you make it "Sorry there aren't any upcoming events scheduled."  I will get a Chinese translation.

Comment (thetwopct) [reply web/wp-content/themes/lfevents/template-parts/upcoming-events-loop.php]: From what I understand the query always returns results (all events) but then filters passed/hidden events as part of the while loop. I've added a simple counter, and if it's 0 results then it displays the no events message.

**Comment by thetwopct** [comment]
OK this is ready for review.

Outstanding issues on my list:
1) The JS resize trigger doesn't seem to be functioning properly once run through webpack build (sigh!), so sometimes (like if you are testing the site by pulling the browser width from mobile to desktop) the dropdown in desktop may suddenly not work.

2) Just realised that the mobile menu needs the triangles after the links to be angled and to move when the menu opens, will implement this tomorrow. The design of this menu is slightly different to the Style Guide but IMO much better, let me know if not OK.

If you are keen to deploy, here is what I did with the menus:

- Delete Linux Foundation link from Home Nav
- Rename Home Nav to Non-Event Nav
- Add new Menu - LF Sites
- Populate with LF sites as per figma
- Link LF Sites menu to LF Sites Nav position
- Delete Nested Menu nav
- Shouldn't need to do anything to LFASIA

Let me know anything else.

**Comment by cjyabraham** [comment]
- [ ] when I go [here](https://pr-608-lfasiallcci.pantheonsite.io/about/about-us/) I can see a light-gray `#fefefe` bar at the bottom of the footer.  Previously this was almost black `#393c41` and I think it looked better as it anchored the footer at the bottom of the browser.  Now the footer looks like it's kind of floating.  Any thoughts on changing it back?
- [ ] the "O"s on the Linux Foundation logo are flattened at the bottom.  If you turn off `height: auto` for the `img` tag then it expands to show them properly.  Can you fix using the appropriate CSS?

**Comment by cjyabraham** [comment]
Nice work.  A few comments:

1. - [ ] when I go [here](https://pr-608-lfasiallcci.pantheonsite.io/about/about-us/) I can see a light-gray `#fefefe` bar at the bottom of the footer.  Previously this was almost black `#393c41` and I think it looked better as it anchored the footer at the bottom of the browser.  Now the footer looks like it's kind of floating.  Any thoughts on changing it back?
2. - [x] the "O"s on the Linux Foundation logo are flattened at the bottom.  If you turn off `height: auto` for the `img` tag then it expands to show them properly rounded.  Can you fix using the appropriate CSS?
3. - [ ] When logged in, the way the "My Profile" menu appears is a different animation to the way the "All Linux Foundation" menu appears.  I think I prefer the "All Linux Foundation" one.  Can we make consistent?
4. - [ ] If I move my browser width to 1044px, "Code of Conduct" and "Contact Us" disappear.  I'm thinking that if some people always use a tablet, they may never see those menu items.  Instead, should we just go to the mobile menu at a wider width
5. - [x] Do you think we still need the words "All Upcoming Events" in the "View All Events" menu of [individual events](https://pr-608-lfeventsci.pantheonsite.io/open-source-summit-japan/)?
6. - [x] The "Create Community Profile" doesn't have the same color transition on hover that the other CTA buttons do on the homepage.  Can this be fixed?
7. - [x] Did you intentionally go with a higher weight on the word "Events" beside the LF logo?  The design says 400 but you have 700 (though I'm now seeing training and lfx have 700...).
8. - [x] From the designs it looks like you need a horizontal line above "All Linux Foundation" in the mobile menu?
9. - [x] The designs show "Create Community Profile" becoming text in the mobile menu but you have kept a button.  Was that intentional?  I think text would be better as it's not a CTA we want to call too much attention to IMO.
10. - [x] Do you think we should drop the external link icon on "Submit a Talk" as it's inconsistent now with all the "All Linux Foundation" links which don't have the icon?
11. - [x] The mobile menu hamburger/"X" is larger than in the style guide.  Is that intentional?
12. - [x] At certain widths (768px) the mobile menu items have larger font (18px) than the word "Events" (16px).  I don't think they should go bigger than (16px) so as not to dwarf it?
13. - [x] If you use your phone in landscape view on the regular events site, you'll notice that the topnav doesn't stick, however, the new one does.  I think it's better for it not to stick when the height is small, as you did with cncf.io.
14. - [x] I know you're almost matching the designs (82px) with the height of the topnav on desktop (80px), however, training (76px) and lfx (55px) use different heights.  I wonder if we should just go with what looks best, in which case I might suggest 76px as it looks a bit higher than it needs to be right now?  Your opinion?
15. - [ ] iPad Pro 11, Safari, landscape orientation, the "All Linux Foundation" down arrow goes below the text:
Screen Shot 2020-11-27 at 2 46 27 PM

**Pull request merged**